Library/Homebrew/cmd/update.sh

```diff
@@ -733,6 +733,20 @@ EOS
     fi
   done
 
+  if [[ -n "${HOMEBREW_INSTALL_FROM_API}" ]]
+  then
+    mkdir -p "${HOMEBREW_CACHE}/api"
+    # TODO: etags?
```

Comment:
Instead of etags, I reckon we can use:

```
curl --header 'If-Modified-Since: Mon, 28 Feb 2022 15:19:26 GMT' https://formulae.brew.sh/api/formula.json
```

with the timestamp of the file